src/Entity/Dataset.php

```diff
@@ -573,7 +573,7 @@ public function getCitation()
         $title = preg_replace('/\.$/', '', $title);
         $udi = $this->getUdi();
         $author = $this->getAuthors();
-        $year = $this->getReferenceDateYear();
+        $year = ($this->getAcceptedDate()) ? $this->getAcceptedDate()->format('Y') : null;
```

Comment:
Maybe add instanceOf ?

Reply:
Maybe we should wait till we decide on what dates to use, and we can probably make changes to all affecting applications in this ticket